src/components/core/geoChart/GeoChartRenderer.tsx

```diff
@@ -232,4 +264,20 @@ export default class GeoChartRenderer extends React.Component<IGeoChartRendererP
         this.cleanupMap();
         this.chart.remove();
     };
+
+    private handlePushpinMoveEnd = (e: mapboxgl.EventData): void => {
+        const { target } = e;
+        const { onCenterPositionChanged } = this.props;
+        const { lng, lat } = target.getCenter();
+        const center: IGeoLngLatObj = { lng, lat };
+        onCenterPositionChanged(center);
```

Comment:
We saved properties like { lng: number, lat: number } to make it will be not confuse, And we will update [number, number] to { lng: number, lat: number } in https://jira.intgdc.com/browse/SD-859